.circleci/config.yml

```diff
@@ -25,10 +26,10 @@ commands:
       - run:
           name: Setup python
           command: |
-            sudo apt-get update && sudo apt-get install -y python3-dev python3-pip python3-venv
-            python3 -mvenv venv
-            . venv/bin/activate
+            sudo apt-get update && sudo apt-get install -y python3-dev python3-pip
             pip3 install flake8 hawkauthlib konfig pyramid pyramid_hawkauth requests simplejson unittest2 WebTest WSGIProxy2
```

Comment:
A system level `pip` would rely on the system level `python`. `pip` is a tiny shell wrapper that basically does the same thing as what you're proposing. Unless you've got a broken install, both `pip` and `python` should be the same in the system path, and if they're not, other things would break. 

I know that poetry subverts all of this and uses it's own mechanisms for installing dependencies, so I think it's fine calling `pip` directly for now.